cylc/flow/config.py

```diff
@@ -710,10 +710,10 @@ def process_initial_cycle_point(self) -> None:
                     icp = ingest_time(orig_icp, get_current_time_string())
                 except IsodatetimeError as exc:
                     raise WorkflowConfigError(str(exc))
-        if orig_icp != icp:
-            # now/next()/previous() was used, need to store
-            # evaluated point in DB
-            self.options.icp = icp
+        # Save un-evaluated point for DB
+        self.raw_icp = None
+        if icp != orig_icp:
+            self.raw_icp = icp
```

Comment:
I feel like the previous comment explained it better?
```suggestion
        self.raw_icp = None
        if icp != orig_icp:
            # now/next()/previous() was used, need to store
            # evaluated point in DB
            self.raw_icp = icp
```